lib/public/views/Logs/Overview/LogsOverviewModel.js

```diff
@@ -365,14 +367,17 @@ export class LogsOverviewModel extends Observable {
     /**
      * Returns the list of URL params corresponding to the currently applied filter
      *
+     * @param {boolean} excludeAnonymous - Whether to exclude anonymous users from the logs.
      * @return {Object} the URL params
      *
      * @private
      */
-    _getFilterQueryParams() {
+    _getFilterQueryParams(excludeAnonymous = false) {
         const sortOn = this._overviewSortModel.appliedOn;
         const sortDirection = this._overviewSortModel.appliedDirection;
 
+        excludeAnonymous && this._authorFilter.update('!Anonymous');
+
```

Comment:
Why do you do that?
You should not update the filters in a function named `get...`
If you want to exclude anonymous by default, the flag should be in the constructor.